pkg/apimachinery/utils/manager.go

```diff
@@ -55,13 +56,13 @@ func ParseManagerKindString(v string) ManagerKind {
 type SourceProperties struct {
 	// The path to the source of the resource.
 	// Can be a file path, a URL, etc.
-	Path string
+	Path string `json:"path,omitempty"`
 
 	// The checksum of the source of the resource.
 	// An example could be a git commit hash.
-	Checksum string
+	Checksum string `json:"checksum,omitempty"`
 
-	// The timestamp of the source of the resource.
+	// The unix millis timestamp of the source of the resource.
 	// An example could be the file modification time.
-	Timestamp time.Time
+	Timestamp int64 `json:"timestamp,omitempty"`
```

Comment:
Let's make sure the resolution is explicit, otherwise one has to guess:
```suggestion
	TimestampMillis int64 `json:"timestampMillis,omitempty"`
```